README.rst

```diff
@@ -20,16 +20,15 @@ Help
 Looking for more help?
 
 - Read `the hosted documentation`_
-- Check `StackOverflow`_ for common problems
```

Comment:
I think this line should be left in

Reply:
No, it's fine like so. But forgot to delete the stackoverflow link and with your question you draw my attention to that - thanks :)